windows/container.go

```diff
@@ -174,6 +174,11 @@ func (c *container) Pid() uint32 {
 	return c.ctr.Pid()
 }
 
+func (c *container) Ps(ctx context.Context) ([]uint32, error) {
+	// TODO (ehazlett): support finding all processes in windows container
```

Comment:
See https://github.com/moby/moby/blob/master/daemon/top_windows.go. That ends up calling into `ProcessList` at https://github.com/moby/moby/blob/master/libcontainerd/client_windows.go#L613 which is satisfied by HCSShim calling `GetComputeSystemProperties` with a process-list query.